src/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx

```diff
@@ -110,10 +148,10 @@ const Metadata: React.FC = props => {
         p={2}
       >
         <Sans size={["3", "4"]} color={["white100", "black100"]}>
-          Gallery Name
+          {name}
         </Sans>
         <Sans size={["3", "4"]} color={["white100", "black100"]}>
-          Closes in 5 days
+          {endAt}
```

Comment:
Dunno if it's of any interest or we already have a way of formatting timestamps in this project? But we could [implement a date formatting field](https://github.com/auspices/atlas/blob/master/app/graphql/shared/timestamps.rb) like there is in Metaphysics so that we could do `endAt(relative: true)` or something.

Reply:
Yeah this has been pretty random across our codebases but in general we've tried to hoist this stuff to MP. I don't (think?) we have a generic formatting field, but that would be nice. I left this as raw output here to remind me to update in MP.

Reply:
[We've got one](https://github.com/artsy/metaphysics/blob/master/src/schema/v2/fields/date.ts) though the format option should be documented with a description and it'd be nice to throw in a relative option too.